src/api/config.rs

```diff
@@ -107,6 +106,8 @@ pub struct EncoderConfig {
   pub show_psnr: bool,
   /// Enables dumping of internal RDO training data.
   pub train_rdo: bool,
+  /// Target CPU feature level.
+  pub cpu_feature_level: crate::cpu_features::CpuFeatureLevel,
```

Comment:
It should not live in a public field. `pub(crate)` please

Reply:
Our benchmark suite really does not like this... Should I conditionally make it `pub` if `bench` feature is enabled? It's kind of ugly...

Reply:
Do not put there the field alternatively.